pkg/aws/ecr_test.go

```diff
@@ -68,6 +73,10 @@ func (m *mockAWSECRClient) DescribeImagesPages(input *ecr.DescribeImagesInput, f
 		m.t.Errorf("Expected repository name to be %s, but was %s", m.expectedRepositoryNames[0], *input.RepositoryName)
 	}
 
+	if input.RegistryId != m.expectedRegistryID {
+		m.t.Errorf("Bad registry ID")
```

Comment:
Maybe use the same error message as used in `DescribeRepositoriesPages`?
```suggestion
 		m.t.Errorf("Expected registry id of %v, but got %v", m.expectedRegistryID, input.RegistryId) 
```

Reply:
Forgot to go back and replace that.  Thanks.